src/LSSOLQPSolver.h

```diff
@@ -69,6 +69,8 @@ class TASKS_DLLAPI LSSOLQPSolver : public GenQPSolver
 	Eigen::VectorXd C_;
 
 	int nrALines_;
+public:
+  EIGEN_MAKE_ALIGNED_OPERATOR_NEW
```

Comment:
Do you know if it's possible to simply declare EIGEN_MAKE_ALIGNED_OPERATOR_NEW in the parent class? It would prevent a lot of repetition...